spec/services/user_updater_spec.rb

```diff
@@ -52,6 +52,9 @@
       expect(IgnoredUser.where(user_id: u2.id).count).to eq 2
       expect(IgnoredUser.where(user_id: u1.id).count).to eq 2
       expect(IgnoredUser.where(user_id: u3.id).count).to eq 0
+      expect(MutedUser.where(user_id: u2.id).count).to eq 2
```

Comment:
Can we check that we have the proper Muted/Ignored user ids instead of just the #?

Reply:
I agree. Then I will also refactor the `MutedUser` specs, not so safe.